docs/source/installation.rst

```diff
@@ -69,25 +67,92 @@ If you're using TypeScript, be sure to reference the declaration file ``excalibu
         </body>
     </html>
 
-Referencing Excalibur via Imports
+Referencing Excalibur via Triple-Slash Reference
+------------------------------------------------
+
+For a simple TypeScript-based game, using triple-slash references works great. It requires
+no extra module system or loaders.
+
+.. code-block:: typescript
+   
+   /// <reference path="node_modules/excalibur/dist/excalibur.d.ts" />
+
+   var game = new ex.Engine({ ... });
+
+Make sure the path is relative to the current TS file. You only need to include the reference
+on your "entrypoint" file. Then simply include ``excalibur.min.js`` as mentioned above in your 
+HTML page. 
+
+You can also reference Excalibur through the ``tsconfig.json``.
+
+.. code-block:: javascript
+
+   {
+      "compilerOptions": {
+         "target": "es5",
+         "outFile": "game.js",
+         "types": ["excalibur"]
+      }
+   }
+
+Referencing Excalibur as a Module
 ---------------------------------
 
-Excalibur does not yet come with out-of-the-box support for ``import`` intellisense in TypeScript. 
-However, until it's natively supported, you can add support yourself with a slight modification.
+Excalibur is built using the `AMD <https://github.com/amdjs/amdjs-api/blob/master/AMD.md>`_ module 
+system. The standalone files ``excalibur.js`` or ``excalibur.min.js`` use the 
+`UMD <https://github.com/umdjs/umd>`_ module syntax at runtime to support CommonJS (Node-like), AMD, and a global browser fallback.  has a self-bootstrapping AMD loader and Excalibur
+is loaded into the ``ex`` global namespace. These are the recommended files to use for production deployments.
 
-Modify your local copy of the distributed ``excalibur.d.ts`` file and add the following to the end:
+You can optionally use ``excalibur.amd.js`` and ``excalibur.amd.d.ts`` to load Excalibur using an
+AMD-compatible loader (such as `jspm <http://jspm.io/>`_).
 
-.. code-block:: typescript
+To get started, first install Excalibur through npm:
 
-   export default ex;
+.. code-block:: bash
+   
+   npm install excalibur -D
 
-This will allow you to reference Excalibur using TypeScript with the import style syntax:
+In a TypeScript project, you can reference Excalibur with the ES6 import style syntax:
 
 .. code-block:: typescript
 
-   import ex from 'excalibur'
+   import * as ex from 'excalibur'
```

Comment:
The old import style was because we had to use "legacy" `export = ex` syntax in declaration files. Now we can use proper ES6 style importing 👍 

This supports `import { Engine } from 'excalibur'` style too.